test/types/signature_with_hash_test.exs

```diff
@@ -0,0 +1,54 @@
+defmodule Kadena.Types.SignatureWithHashTest do
+  @moduledoc """
+  `SignatureWithHash` struct definition tests.
+  """
+
+  use ExUnit.Case
+
+  alias Kadena.Types.{
+    Signature,
+    SignatureWithHash,
+    SignedSignatureWithHash,
+    UnsignedSignatureWithHash
+  }
+
+  describe "new/1" do
+    test "with a valid SignedSignatureWithHash" do
+      signed_signature =
+        SignedSignatureWithHash.new(
+          hash: "valid_hash",
+          sig: Signature.new("valid signature"),
+          pub_key: "public_key"
```

Comment:
@santiagoboterokommit Please use real hashes, signatures, and public keys. Apply the same suggestion to the other test cases.